src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -84,10 +94,38 @@ class Submitter(stagingBaseDir: File,
     _ <- isValid(stageBag)
     // EASY-1464 3.3.7 checksums
     _ <- samePayloadManifestEntries(stageBag, draftBag)
+    _ <- setRightsRecursively(stageBag.baseDir.parent)
     // EASY-1464 step 3.3.9 Move copy to submit-to area
     _ = stageBag.baseDir.parent.moveTo(submitDir)
   } yield ()
 
+  private def setRightsRecursively(file: File): Try[Unit] = {
+    resource.managed(Files.walk(file.path, Int.MaxValue, VisitOptions.default: _*))
+      .map(_.iterator().asScala.map(setRights).find(_.isFailure).getOrElse(Success(())))
+      .tried
+      .flatten
+  }
+
+  private def setRights(path: Path): Try[Unit] = Try {
+    trace(path)
+    // EASY-1932, variant of https://github.com/DANS-KNAW/easy-split-multi-deposit/blob/73189001217c2bf31b487eb8356f76ea4e9ffc31/src/main/scala/nl.knaw.dans.easy.multideposit/actions/SetDepositPermissions.scala#L72-L90
+    File(path).addPermission(GROUP_WRITE)
+    // tried File(path).setGroup(groupPrincipal) but it causes "java.io.IOException: 'owner' parameter can't be a group"
+    Files.getFileAttributeView(
+      path,
+      classOf[PosixFileAttributeView],
+      LinkOption.NOFOLLOW_LINKS,
+    ).setGroup(groupPrincipal)
+  }.recoverWith {
```

Comment:
Could all these exceptions just be called `e` or will that result in a naming conflict? I agree that a long name is not necessary, but names like `upnf`, though fun to pronounce, don't add any information either.